### DISCUSSION TITLE
Standup (Thursday, November 3rd)

### DISCUSSION THREAD

**Discussion by DavBot02** [BBGTC/food-bank-app#48] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on color palette and components
 - Looked into a linter for the project

**What are your priorities for the next 24 hours?**:
 - Add linter (if it's worth it)

**Blockers**:
 - I'm too used to React and some stuff works different

**Shoutouts**:
 - @CarlosCRG19 for his project

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Made first tests!
 - Started working on pipeline 

**What are your priorities for the next 24 hours?**:
 - PR with tests and pipeline (#38 
 - Maybe help Brandon with #37 
 - Work on #39 

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02 for being lazy

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Opened #46 

**What are your priorities for the next 24 hours?**:
 - Start working on frontend architecture and DB seeding

**Blockers**:
 - Algorithms homework

**Shoutouts**:
 - @everyone you guys are phenomenal

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Worked on #37 
 -  Discussed with @antonieto about backend structure

**What are your priorities for the next 24 hours?**:
 -  Finish #37  and open up PR
 - Define what changes are we going to include considering Brenda's retro
 
**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02 because he is a hottie
